gnomad_qc/v4/annotations/vrs_annotation_batch.py

```diff
@@ -372,20 +359,18 @@ def main(args):
 
         # Checkpoint (write) resulting annotated table
         ht_annotated = ht_annotated.checkpoint(
-            f"gs://gnomad-vrs-io-finals/ht-outputs/annotated-checkpoint-VRS-{prefix}.ht",
+            v4_vrs_annotations(test=args.test).path,
             overwrite=args.overwrite,
         )
         logger.info(
             "Annotated Hail Table checkpointed to:"
-            f" gs://gnomad-vrs-io-finals/ht-outputs/annotated-checkpoint-VRS-{prefix}.ht"
+            f" {v4_vrs_annotations(test=args.test).path}"
         )
 
     if args.annotate_original:
         check_resource_existence(
```

Comment:
add:
output_path = v4_vrs_annotations(test=args.test, original_annotations=True)
and then replace any references to "output_paths_dict[version]" and remove the output_paths_dict dictionary